docs/data/sdks/ios-swift/index.md

```diff
@@ -97,13 +97,25 @@ amplitude.identify(identify: identify)
 
 --8<-- "includes/groups-intro-paragraph.md"
 
-For example, if Joe is in 'orgId' '10' and '16', then the `groupName` would be '[10, 16]'). Here is what your code might look like:
-
 ```swift
 amplitude.setGroup(groupType: "orgId", groupName: "15")
 amplitude.setGroup(groupType: "sport", groupName: ["tennis", "soccer"])
 ```
 
+--8<-- "includes/event-level-groups-intro.md"
+
+```swift
+amplitude.track(
+    event: BaseEvent(
+        eventType: "event type",
+        groups: ["orgId": 15]
+    ),
+    eventProperties: [
+        "eventPropertykey": "eventPropertyValue"
+    ]
+)
```

Comment:
```suggestion
amplitude.track(
    event: BaseEvent(
        eventType: "event type",
        groups: ["orgId": 15],
        eventProperties: [
            "eventPropertykey": "eventPropertyValue"
        ]
    ),
)
```

Reply:
I think it's also to point out the type of the BaseEvent in each SDKs doc to let them know what they can set.

Reply:
Fixed. Put event properties before groups as "Argument 'eventProperties' must precede argument 'groups'"
```
amplitude.track(
    event: BaseEvent(
        eventType: "event type",
        eventProperties: [
            "eventPropertykey": "eventPropertyValue"
        ], 
        groups: ["orgId": 15]
    )
)
```